java/src/jmri/jmrit/beantable/signalmast/SignalMastTableDataModel.java

```diff
@@ -1,24 +1,40 @@
 package jmri.jmrit.beantable.signalmast;
 
+import java.awt.Point;
+import java.awt.Rectangle;
+import java.awt.Component;
+import java.awt.BorderLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseEvent;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.EventObject;
 import java.util.Hashtable;
 import java.util.Vector;
 import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
+import javax.swing.DefaultCellEditor;
 import javax.swing.JButton;
 import javax.swing.JComboBox;
 import javax.swing.JTable;
+import javax.swing.JPanel;
 import javax.swing.JTextField;
-import javax.swing.RowSorter;
+import javax.swing.ListCellRenderer;
+import javax.swing.SwingUtilities;
 import javax.swing.table.TableCellEditor;
 import javax.swing.table.TableCellRenderer;
+import javax.swing.table.TableColumn;
 import javax.swing.table.TableModel;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+
 import jmri.InstanceManager;
 import jmri.Manager;
 import jmri.NamedBean;
 import jmri.SignalMast;
 import jmri.jmrit.beantable.BeanTableDataModel;
-import jmri.jmrit.beantable.SignalMastTableAction.MyComboBoxEditor;
-import jmri.jmrit.beantable.SignalMastTableAction.MyComboBoxRenderer;
+import jmri.jmrit.beantable.SignalMastTableAction.MyComboBoxEditor; // @Deprecated
```

Comment:
Removed my comment, as well as the unused imports that I relied on during development.